.github/workflows/auto_updates.yml

```diff
@@ -0,0 +1,62 @@
+# Workflow for updating Rust dependencies with Cargo and Javascript dependencies with npm.
+# Major version updates are handled separately, by Dependabot.
+---
+name: Update Deps
+
+on:
+  workflow_dispatch:
+  # Run every Monday at 0530 UTC
+  schedule:
+    - cron: '30 5 * * 1'
+
+jobs:
+  workflow-auto-updates:
+    name: Update dependencies
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+
+      - name: Set up Node
+        uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c # v3.6.0
+        with:
+          node-version: 18
```

Comment:
A fixed node version here will probably cause maintenance overhead? Isn't node installed on every runner anyway?

Reply:
> Isn't node installed on every runner anyway?

Node is [installed and cached on the Ubuntu 20.04 runners](https://github.com/actions/runner-images/blob/ubuntu20/20230710.1/images/linux/Ubuntu2004-Readme.md#nodejs), but the [documentation for the `actions/setup-node` action](https://github.com/actions/setup-node) states:

> The `node-version` input is optional. If not supplied, the node version from PATH will be used. However, it is recommended to always specify Node.js version and don't rely on the system one.

The idea was to specify a version that would use a cached version of node that is already available on the runner because that is quick...only 2 seconds to run the "Set up Node" step:

<img width="979" alt="image" src="https://github.com/phylum-dev/cli/assets/18729796/f4a29da8-122e-429e-9b38-4647fc27e74e">

A more detailed reading of the `actions/setup-node` action documentation shows that there are options to satisfy our shared desires:

* Setting the value for `node-version` to `*` will "get the latest locally-cached Node.js version"
* `node-version-file` (e.g., `.node-version`) can be used to specify the node version instead of using `node-version`
  * That might reduce the maintenance overhead by providing a single reference source
  * `website/package.json` could be used since it already already contains the `engines.node` value [as requested in the docs](https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#node-version-file)

Reply:
> node-version-file (e.g., .node-version) can be used to specify the node version instead of using node-version 

That's not what we want though. We want to always update it automatically so we don't have to worry about it. It's a tiny project, it should never need any maintenance intervention.

The node version installed on the runner should always work.

Reply:
Sounds like a vote for setting the value for `node-version` to `*`. Change coming.